From 376a13e7c41cc5528b6088d0dd16ec1b323a8d37 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 29 Jul 2026 18:12:11 +0200
Subject: [PATCH 14/N] README/VERSION: advertise that InnoSetup v7.0.2 is
 supported, too

I forgot to bump this in the previous commits.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 README.md | 4 ++--
 VERSION   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/README.md b/README.md
index 3a5e826..cdfc2bf 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 
-# innoextract - A tool to unpack installers created by Inno Setup
 
-[Inno Setup](https://jrsoftware.org/isinfo.php) is a tool to create installers for Microsoft Windows applications. innoextract allows to extract such installers under non-Windows systems without running the actual installer using wine. innoextract currently supports installers created by Inno Setup 1.2.10 to 6.3.3.
+
+[Inno Setup](https://jrsoftware.org/isinfo.php) is a tool to create installers for Microsoft Windows applications. innoextract allows to extract such installers under non-Windows systems without running the actual installer using wine. innoextract currently supports installers created by Inno Setup 1.2.10 to 7.0.2.
 
 In addition to standard Inno Setup installers, innoextract also supports some modified Inno Setup variants including Martijn Laan's My Inno Setup Extensions 1.3.10 to 3.0.6.1 as well as GOG.com's Inno Setup-based game installers. innoextract is able to unpack Wadjet Eye Games installers (to play with AGS), Arx Fatalis patches (for use with Arx Libertatis) as well as various other Inno Setup executables.
 
diff --git a/VERSION b/VERSION
index e2b4fe9..e42b64b 100644
--- a/VERSION
+++ b/VERSION
@@ -1,7 +1,7 @@
 innoextract 1.10-dev
 
 Known working Inno Setup versions:
-Inno Setup 1.2.10 to 6.3.3
+Inno Setup 1.2.10 to 7.0.2
 
 Bug tracker:
 https://innoextract.constexpr.org/issues
